frontends/web/src/routes/router.tsx

```diff
@@ -151,6 +151,7 @@ export const AppRouter = ({ devices, deviceIDs, devicesKey, accounts, activeAcco
 
   const ManageBackupsEl = <InjectParams><ManageBackups
     key={devicesKey('manage-backups')}
+    deviceID=""
```

Comment:
this confused me a bit, maybe add a comment that says the `InjectParams` will override/correctly set it? But maybe its not a good idea, to add too many comments.

we could define a `const TO_INJECT = ""` with a comment, and then always use that in such situations? no strong opinion, feel free to ignore this.

Reply:
Indeed this is confusing. I changed this to how other route components are initialized (with `deviceID={null})`.

In the future I think we should take a look if we can simplify router.tsx and get rid of `InjectParams`.